terraform/modules/tf_stream_alert/metrics.tf

```diff
@@ -13,21 +13,6 @@ resource "aws_cloudwatch_log_metric_filter" "rule_processor_cw_metric_filters" {
   }
 }
 
-// CloudWatch metric filters for the alert processor
```

Comment:
Mabye I'm missing something, but is this in the new set of modules?

Reply:
It is not - there are no custom metrics in the alert processor, so this resource is never used. I'm removing it for now - we can always re-add it if we ever do put custom metrics in the alert processor